auth-service/README.md

```diff
@@ -179,6 +179,36 @@ curl 'http://localhost:8088/auth-internal/bearer/user-info' \
 
 The response is similar to above, though the access token is not repeated.
 
+### Authenticating with the Delta API
+
+#### Obtaining an API token
+
+This endpoint requires authenticating with a valid AD username and password, and also a valid client id and secret combination. If running locally an id and secret must be added manually to the database. The test auth service DB contains `dev` and `dev` as an ID and secret pair. The stage auth service DB contains an id `dev-client` and a secret that can be found in AWS by the name `auth-service-dev-api-client-secret-staging`. When testing on test or stage, the domain should be replaced as appropriate. The origin domain should be `api.delta.[environment].communities.gov.uk`.
+
+```sh
+curl --request POST 'http://localhost:8088/delta-api/oauth/token' \
+--header 'Content-Type: application/x-www-form-urlencoded' \
+--header 'Origin: http://localhost:8080' \
+--data-urlencode 'grant_type=password' \
+--data-urlencode 'client_id=<client-id>' \
+--data-urlencode 'client_secret=<client-secret>' \
+--data-urlencode 'username=<username>' \
+--data-urlencode 'password=<password>'
+```
+
+#### Exchanging an API token
+
+Once obtained, the API token can be exchanged for a SAML token:
```

Comment:
And mention the fact that this is normally a Delta API -> Auth service request, not directly from the local authority or swagger.